backend/server/server.go

```diff
@@ -648,6 +647,22 @@ func getInitSetting(ctx context.Context, datastore *store.Store) (*workspaceConf
 		return nil, err
 	}
 
+	// initial workspace general setting
+	bytes, err := json.Marshal(storepb.WorkspaceGeneralSettingPayload{
```

Comment:
We want to use protojson consistently.

https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/backend/store/db_schema.go?L59:13&subtree=true